_episodes/03-skimage-images.md

```diff
@@ -387,11 +313,9 @@ extraneous background detail has been removed.
 >
 > ![Su-Do-Ku puzzle](../data/sudoku.png)
 >
-> Navigate to the `code/03-skimage-images/`
-> directory, and copy the `HighIntensity.py` program to another file named
-> `LowIntensity.py`. Then, edit the `LowIntensity.py` program so that it
-> turns all of the white pixels in the image to a light gray color, say with
-> all three color channel values for each formerly white pixel set to 64. Your
+> Replicate the ignore low intensity script above as a starting. Then, edit the script 
```

Comment:
```suggestion
> As a starting point, replicate the code we used to ignore low intensity pixels above. Then, edit this copy 
```

Reply:
This one is not longer relevant after picking up changes from gh-pages in previous commit